addons/weather/XEH_postInit.sqf

```diff
@@ -11,13 +11,14 @@ GVAR(wind_period_start_time) = CBA_missionTime;
 if (!isServer) then {
     "ACE_MISC_PARAMS" addPublicVariableEventHandler {
         TRACE_1("MISC PARAMS PVEH",ACE_MISC_PARAMS);
+        GVAR(currentOvercast) = (ACE_MISC_PARAMS select 0);
```

Comment:
I don't know why there are `()` on the right side in the rest of the params, but they are really pointless.

Reply:
We can fix it in the following cleanup PRs.

Reply:
This only runs on non server's
So GVAR(currentOvercast) is always 0 on server?

Reply:
The server never uses:
```GVAR(currentTemperature), GVAR(currentHumidity) and GVAR(currentOvercast)```